### DISCUSSION TITLE
what exactly does `:hint` hint?

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#7711] [Q&A] [answered]
I want to add a custom hinting group for certain websites (e.g. `:hint example`), but have the fallback behaviour (for websites i haven't manually defined) hint like `:hint` does. I've got my idea mostly working with setting `hints.selectors` first for every site, and then overwriting it for each site i want to have custom ones. `:hint example` hints the correct stuff on site A and B, and will do a different set of hints on any other site.

My issue is I can't seem to find a list of what selectors `:hint` uses, to try and replicate the things it hints. I have looked around pretty extensively, and I'm sure it's listed *somewhere*, but I've had a hard time finding it.

Thanks for the patience for a relatively simple question :) apologies if i'm misusing terminology or calling stuff by the wrong names, i'm not the most familiar with configuring qutebrowser.

**Comment by toofar** [comment, marked as answer]
I'm not a heavy customise of hints so I might be missing something about what you are trying to do. But to me it looks like:
* they are in the docs for the setting: https://qutebrowser.org/doc/help/settings.html#hints.selectors
* that setting support URL patterns, so you should be able to override it per site while leaving the global settings at the default (which you might already be doing)
* in config.py the setting should be able to be accessed like normal python objects (a dict of lists in this case) so I think you should be able to just append any additional selectors without having to duplicate the default list, if you want to do that

**Comment by ghost** [comment]
...its just clicked for me that i should just modify the "all" selector instead of making my own custom one that sometimes acts like "all", you're 100% right. thanks for the help :)
